src/components/_molecules/TagsInput/TagsInput.component.tsx

```diff
@@ -143,6 +142,23 @@ const TagsInput: React.FC<TagsInputProps> = (
     [addTag, removeTag, inputValue, spaceSeparates]
   );
 
+  const onPaste = useCallback(
+    (e: React.ClipboardEvent) => {
+      const pastedValue = e.clipboardData.getData('text');
+
+      if (pastedValue.includes(' ')) {
+        const splitPastedValue = pastedValue
+          .split(/(\s+)/)
```

Comment:
In the ticket acceptance criteria is expected to also have comma as the email separator.
We just need to add the **,** to the regex.

```suggestion
          .split(/[\s,]+/)
```

Reply:
Right, all clear... Changed, added comma as the separator.